src/pages/AddServers/CompleteInstallOS.js

```diff
@@ -0,0 +1,83 @@
+// (c) Copyright 2018 SUSE LLC
+/**
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+**/
+
+import React from 'react';
+import BaseUpdateWizardPage from '../BaseUpdateWizardPage.js';
+import { LoadingMask } from '../../components/LoadingMask.js';
+import { translate } from '../../localization/localize.js';
+import { alphabetically } from '../../utils/Sort.js';
+
+// This is the complete page for installing SLES OS on compute servers
+// It will display server ID, IP, mac-addr and ilo-ip
+class CompleteInstallOS extends BaseUpdateWizardPage {
+  constructor(props) {
+    super(props);
+    this.state = {
+      // loading errors from wizard model or progress loading
+      wizardLoadingErrors: this.props.wizardLoadingErrors,
```

Comment:
I'm surprised these cant just be referenced off of the props object, do they have to be part of the state? they arent changed within this class, applies to the other classes that use this as well, maybe I'm just react rusty...

Reply:
let me try if I can get rid of some code